tuxemon/resources/l18n/base.pot

```diff
@@ -0,0 +1,83 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Is this file still required?

Reply:
Yes and no. This file is the PO template which can be copy-pasted and populated with `slug`s or used directly to generate language-specific PO files.

Reply:
If its not required by the gettext library, I think any template for translations would either
* Use the English PO
* Use a script to generate template base on slugs in the database and maps

Keeping a template around seems like it would require some upkeep to make sure it is in sync with the database.  I'd like to see it removed if not absolutely needed.